install/installer/pkg/components/image-builder-mk3/deployment.go

```diff
@@ -81,6 +81,7 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 			VolumeSource: corev1.VolumeSource{
 				Secret: &corev1.SecretVolumeSource{
 					SecretName: secretName,
+					Items:      []corev1.KeyToPath{{Key: ".dockerconfigjson", Path: "pull-secret.json"}},
```

Comment:
nit: Is `pull-secret` the right name? as we would also use it for pushes here. Could be tackled with https://github.com/gitpod-io/security/issues/89

Reply:
Its name is `pull-secret.json` now, ref [code](https://github.com/gitpod-io/gitpod/blob/017cfbfbec99f6d8480a91c7c6be98ac8ad7d6a5/install/installer/pkg/components/image-builder-mk3/constants.go#L10).